Cargo.toml

```diff
@@ -9,6 +9,10 @@ readme = "README.md"
 categories = ["wasm"]
 documentation = "https://rustwasm.github.io/wasm-pack/"
 
+[workspace]
+members = ["binary-install"]
```

Comment:
Since there's a dependency on `binary-install` I think this line can be removed

Reply:
Ah yes!